samples/env

```diff
@@ -0,0 +1,27 @@
+sudo virsh start edpm-compute-0
+crc start
+eval $(crc oc-env)
+oc login -u kubeadmin -p 12345678 https://api.crc.testing:6443  
+
+export ADMIN_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' AdminPassword:' | awk -F ': ' '{ print $2; }')
```

Comment:
also can yo ufafctor this out into a variable

i had script that assume they can put stuff in my home dir arbitarally

Reply:
do you mean the kubeadmin password?

Reply:
the intent although is to keep those the same as in docs, assuming the same paths used in both places. Not automating variables to allow fully customized runs (there are test suits for that)